lib/src/test/java/org/exoplatform/social/addons/storage/dao/SpaceMemberDAOTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2003-2016 eXo Platform SAS.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.exoplatform.social.addons.storage.dao;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.exoplatform.social.addons.storage.entity.AppEntity;
+import org.exoplatform.social.addons.storage.entity.IdentityEntity;
+import org.exoplatform.social.addons.storage.entity.SpaceEntity;
+import org.exoplatform.social.addons.storage.entity.SpaceMemberEntity;
+import org.exoplatform.social.addons.storage.entity.SpaceMemberEntity.Status;
+import org.exoplatform.social.addons.test.BaseCoreTest;
+import org.exoplatform.social.core.identity.provider.OrganizationIdentityProvider;
+import org.exoplatform.social.core.identity.provider.SpaceIdentityProvider;
+
+public class SpaceMemberDAOTest extends BaseCoreTest {
```

Comment:
Good catch! I thought it was added automatically, but yes, no @Test annotation is used neither.
Done.